.github/workflows/integration-tests.yaml

```diff
@@ -295,3 +295,49 @@ jobs:
         with:
           name: tests-logs
           path: artifacts
+
+  integration-tests-legacy-v2:
+    name: Integration Tests Legacy V2
+    runs-on: ubuntu-20.04
+    needs: prepare-integration-tests
+    steps:
+      - name: Apicurio Registry (Version 2.5.x) Checkout
+        run: |
+          mkdir registry-v2
+          cd registry-v2
+          git init
+          git config --global user.name "apicurio-ci"
+          git config --global user.email "apicurio.ci@gmail.com"
+          git remote add origin "https://apicurio-ci:${{ secrets.ACCESS_TOKEN }}@github.com/Apicurio/apicurio-registry.git"
+          git fetch --all
+          git checkout 2.5.x
+          git branch --set-upstream-to=origin/2.5.x
+          git pull
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Run Legacy Integration Tests (Core API v2)
+        run: |
+          echo "Starting Registry App (In Memory)"
+          docker run -it -p 8181:8080 -d --env CORS_ALLOWED_ORIGINS=* ttl.sh/${{ github.sha }}/apicurio/apicurio-registry:1d
+          cd registry-v2
+          ./mvnw -T 1.5C -Pintegration-tests clean install -DskipTests=true -Dmaven.javadoc.skip=true
```

Comment:
If I read it correctly this can be simplified with something like:

```bash
./mvnw -T 1.5C verify -am -pl integration-tests -Pregression -Dmaven.javadoc.skip=true -Dquarkus.http.test-host=localhost -Dquarkus.http.test-port=8181
```

But I found out that the `integration-tests` are not included in the root POM modules ...

Reply:
That will work, but it will also run all of the unit tests in the other modules.  What I have is faster because it will skip all tests and whatnot when building everything, and then **only** run the integration tests, directly from the `integration-tests` module.  I have found that even if I've already built the entire project, running the integration tests from the root with your command will still run unit tests from some of the other modules.